manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/services/AbstractConnectorService.java

```diff
@@ -106,8 +122,8 @@ private Connector doCreateConnector(Bridge bridge, String customerId, String own
     }
 
     @Override
-    public ConnectorResponse toResponse(Connector connector) {
-        ConnectorResponse connectorResponse = generateSpecificResponse(connector);
+    public T toResponse(Connector connector) {
```

Comment:
The problem is... now that you've implemented retrieval... that in `v1` we masked the "secret" Managed Connector parameters (such as passwords and tokens etc) using `GatewaySecretsHandler`. Your implementation in `v2` returns the plain values. This will have an impact on `PUT` too (that I suspect you'll be doing after the easier `DELETE`).

I'm not proposing that this PR be stalled.. but you might need to create a JIRA to ensure it's handled at some point.

Reply:
https://issues.redhat.com/browse/MGDOBR-1410